test/e2e/data/infrastructure-docker/main/clusterclass-quick-start-runtimesdk-v1beta1.yaml

```diff
@@ -0,0 +1,152 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
Basically identical (apart from the name) to the release-1.10 version of test/e2e/data/infrastructure-docker/main/clusterclass-quick-start-runtimesdk.yaml

(also a v1beta1 version of test/e2e/data/infrastructure-docker/main/clusterclass-quick-start-runtimesdk.yaml)